yt_dlp/extractor/ninenow.py

```diff
@@ -59,26 +55,36 @@ def _real_extract(self, url):
             cache = page_data.get(kind, {}).get('%sCache' % kind, {})
             if not cache:
                 continue
-            common_data = (cache.get(current_key) or list(cache.values())[0])[kind]
+            common_data = {
+                'episode': (cache.get(current_key) or list(cache.values())[0])[kind],
+                'season': (cache.get(current_key) or list(cache.values())[0]).get('season', None)
+            }
             break
         else:
             raise ExtractorError('Unable to find video data')
 
-        video_data = common_data['video']
-
-        brightcove_id = video_data.get('brightcoveId') or 'ref:' + video_data['referenceId']
-        video_id = compat_str(video_data.get('id') or brightcove_id)
-
-        if not self.get_param('allow_unplayable_formats') and video_data.get('drm'):
-            self.report_drm(video_id)
-
-        title = common_data['name']
+        # Video Data extraction
+        if try_get(common_data, lambda x: x['episode']['video']['drm'], bool):
+            raise ExtractorError('This video is DRM protected.', expected=True)
```

Comment:
```suggestion
        if not self.get_param('allow_unplayable_formats') and try_get(common_data, lambda x: x['episode']['video']['drm']):
            self.report_drm(video_id)
```

Reply:
There isn't a docstring on `report_drm()` what should the `video_id` be here?

Reply:
It is used only to print the message. So either video_id/display_id is fine. But since you are using the display_id in the other messages, use the same